redfish-core/lib/ethernet.hpp

```diff
@@ -1103,20 +1103,91 @@ inline void
         dbus::utility::DbusVariantType(macAddress));
 }
 
+void deleteIPv4StaticAddresses(
+    const std::string& ethifaceId,
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp, ethifaceId](const boost::system::error_code errorCode,
+                                dbus::utility::ManagedObjectType& resp) {
+        if (errorCode)
+        {
+            messages::internalError(asyncResp->res);
+        }
+        const std::string ipPathStart =
+            "/xyz/openbmc_project/network/" + ethifaceId;
+
+        for (const auto& objpath : resp)
+        {
+            if (objpath.first.str.starts_with(ipPathStart + "/"))
+            {
+                for (const auto& interface : objpath.second)
+                {
+                    if (interface.first == "xyz.openbmc_project.Network.IP")
+                    {
+                        auto type = std::find_if(interface.second.begin(),
+                                                 interface.second.end(),
+                                                 [](const auto& property) {
+                            return property.first == "Type";
+                        });
+                        const std::string* typeStr =
+                            std::get_if<std::string>(&type->second);
+
+                        if (typeStr == nullptr ||
```

Comment:
upstream more common to throw an error if nullptr (it should not be nullptr so something terrible happened if nullptr)

Reply:
in this case its just about reading property value to check if its null, it just ignores and continue to check other objects.
same followed in the existing.. I don't anything terrible can happen with bmcweb if this property ptr is null
https://github.com/openbmc/bmcweb/blob/master/redfish-core/lib/ethernet.hpp#L456